packages/vitest/src/node/cli.ts

```diff
@@ -112,9 +112,6 @@ async function run(cliFilters: string[], options: UserConfig) {
   }
   finally {
     if (!ctx.config.watch)
-      await ctx.close()
+      await ctx.exit()
```

Comment:
Because this was `ctx.close()` all timeout logic was not doing anything anyway :)